src/uwtools/api/config.py

```diff
@@ -107,6 +108,7 @@ def realize(
     _realize(
         input_config=_ensure_config_arg_type(input_config),
         input_format=input_format,
+        output_block=output_block,
```

Comment:
The docstrings for `realize()` and `realize_to_dict()` need to be updated to include the new `output_block` argument. At the moment, this argument is undocumented in the API docs.